.changeset/odd-wasps-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+'@techdocs/cli': minor
+---
+
+OpenStack Swift configuration changed due to OSS SDK Client change in @backstage/techdocs-common, it was a breaking change.
```

Comment:
Maybe add a reference to the PR https://github.com/backstage/backstage/pull/6839?

Reply:
done that :)